### ISSUE TITLE
error in plugin with latest beta

### ISSUE THREAD

**Issue by sander816** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#62]
plugin seems to work fine, but i see these errors in domoticz.log:
```

2019-01-05 13:48:35.460 (zigbee network) Zigbee2mqtt bridge is online
2019-01-05 13:48:35.460 (zigbee network) Joining new devices is disabled on the zigbee bridge
2019-01-05 13:48:35.512 (zigbee network) Received available devices list from bridge
2019-01-05 13:48:35.512 (zigbee network) Device RTCGQ11LM 0x00158d00029bff93 (0x00158d00029bff93)
2019-01-05 13:48:35.083 Status: (zigbee network) Stopping threads.
2019-01-05 13:48:35.083 Status: (zigbee network) Stopped.
2019-01-05 13:48:35.084 Status: (zigbee network) Started.
2019-01-05 13:48:35.303 Status: (zigbee network) Initialized version 0.0.15
2019-01-05 13:48:35.303 Status: (zigbee network) Entering work loop.
2019-01-05 13:48:39.055 (smart meter) P1 Smart Meter (Electriciteit)
2019-01-05 13:48:39.059 HttpLink: sending global variable huidig_stroomverbruik with value: 173.0
2019-01-05 13:48:39.079 HttpLink: response
2019-01-05 13:48:39.081 (smart meter) Usage (Usage L1)
2019-01-05 13:48:39.084 (smart meter) Usage (Delivery L1)
2019-01-05 13:48:44.631 Error: (zigbee) 'onHeartbeat' failed 'AttributeError':''NoneType' object has no attribute 'onHeartbeat''.
2019-01-05 13:48:44.632 Error: (zigbee) ----> Line 191 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onHeartbeat
2019-01-05 13:48:44.632 Error: (zigbee) ----> Line 113 in '/home/pi/domoticz/plugins/zigbee2mqtt/plugin.py', function onHeartbeat
2019-01-05 13:48:49.030 (smart meter) P1 Smart Meter (Electriciteit)

```

**Comment by stas-demydiuk** [comment]
Do you have 2 instances of plugin running at the same time: `zigbee network`, `zigbee`?

**Comment by sander816** [comment]
no, not that i know.
I enabled the plugin again, and the error is gone

Op ma 7 jan. 2019 om 11:49 schreef Stanislav Demydiuk <
notifications@github.com>:

**Comment by stas-demydiuk** [comment]
Ok, please reopen the ticket if you will be able to reproduce the issue.

**Comment by JHurk** [comment]
I have this (or similar) error after upgrading the plugin to 0.2.0

2020-05-03 18:35:19.615 Status: Incoming connection from: 192.168.1.1
2020-05-03 18:35:25.924 Error: (Zigbee2MQTT) 'onHeartbeat' failed 'AttributeError':''NoneType' object has no attribute 'onHeartbeat''.
2020-05-03 18:35:25.924 Error: (Zigbee2MQTT) ----> Line 235 in '/usr/local/domoticz/var/plugins/domoticz-zigbee2mqtt-plugin-master/plugin.py', function onHeartbeat
2020-05-03 18:35:25.924 Error: (Zigbee2MQTT) ----> Line 107 in '/usr/local/domoticz/var/plugins/domoticz-zigbee2mqtt-plugin-master/plugin.py', function onHeartbeat

I have tried deactivating and activating the plugin again, but the error returned right after activating the plugin.

I have Domoticz running as Synology Package with the Zigbee2MQTT-plugin and the MQTT-plugin.
I upgraded as followed (it is a little unconventional therefor I mention it):
- I downloaded the new version on my Macbook and unpacked it
- I copied the unpacked folder to the plugin folder of Domoticz (same foldername so overwriting I guess, could not choose)
- Stopped the package of Domoticz and tried to restart is, this didn't work (happens more often)
- I restarted my NAS, Domoticz worked again but it looked like the old version (no new UI)
- Deleted the folder domoticz-zigbee2mqtt-plugin-master in the plugin directory
- Copied the ZIP-file to the plugin folder on my NAS and unpacked it there (deleted the ZIP-file afterwards)
- Restarted Domoticz (this time it did work)
- Still the old version (no UI) so de-activated the plugin and activated it again, this did not change anything. Looked in the log and saw the errors.
